airbyte-integrations/bases/base-normalization/integration_tests/normalization_test_output/postgres/test_simple_streams/dbt_project.yml

```diff
@@ -15,7 +15,7 @@ profile: 'normalize'
 # These configurations specify where dbt should look for different types of files.
 # The `source-paths` config, for example, states that source models can be found
 # in the "models/" directory. You probably won't need to change these!
-source-paths: ["models"]
+source-paths: ["modified_models"]
```

Comment:
maybe this file somehow hasn't been regenerated for a long time? this looks like a really old change https://github.com/airbytehq/airbyte/blame/master/airbyte-integrations/bases/base-normalization/integration_tests/test_normalization.py#L253

Reply:
models and modified_models should be both versioned in git.

modified_models tracks changes made to generated SQL if the catalog.json is being changed between syncs.
it's a test that is a bit early since we are not supposed to handle schema evolution yet...